src/quantum/config.js

```diff
@@ -535,6 +583,41 @@ const TP6_SITES_DATA = {
+    ['Chrome', 'warm', 'Tp6: Apple', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-apple-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Microsoft', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-microsoft-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Office', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}},{eq: {suite: 'raptor-tp6-office-chromium'}}]}],
+    ['Chrome', 'warm', 'Tp6: Reddit', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-reddit-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Docs', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-docs-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: eBay', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-ebay-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Google Mail', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-google-mail-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Instagram', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-instagram-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: PayPal', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-paypal-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Pinterest', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-pinterest-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Sheets', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-sheets-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Slides', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-slides-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Tumblr', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-tumblr-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Twitter', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-twitter-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Twitch', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-twitch-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Wikipedia', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-wikipedia-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Yahoo Mail', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-yahoo-mail-chrome'}}]}],
+    ['Chrome', 'warm', 'Tp6: Yahoo News', {and: [{gte: {push_timestamp: {date: '2019-09-01'}}}, {eq: {suite: 'raptor-tp6-yahoo-news-chrome'}}]}],
+
+    ['Chrome', 'cold', 'Tp6: Facebook', {eq: {suite: 'raptor-tp6-facebook-chrome-cold'}}],
+    ['Chrome', 'cold', 'Tp6: Amazon', {eq: {suite: 'raptor-tp6-amazon-chrome-cold'}}],
+    ['Chrome', 'cold', 'Tp6: Google', {eq: {suite: 'raptor-tp6-google-chrome-cold'}}],
+    ['Chrome', 'cold', 'Tp6: YouTube', {eq: {suite: 'raptor-tp6-youtube-chrome-cold'}}],
```

Comment:
I'm guessing we don't have an issue with '-chromium' and '-chrome' data existing for the chromium cold-load page tests?

Reply:
@gmierz yes, the cold pageload never had this problem.  This can demonstrated with a full-year cold load time view:  https://deploy-preview-566--firefox-health-dashboard.netlify.com/quantum/tp6?test=cold-loadtime&platform=win64&past=year&ending=2019-09-23